### PR TITLE
Add Timestamps to Extension Sync Messages

### LINKED ISSUE #2554: [Enhancement] TAE: Show "Synced" messages with the timestamp
The goal of this task is showing the success message following the same standard we have for themes:

image

**Stretch goal**: it would be awesome if we could also list the updated files, as the following snippet shows. However, we would need to implement an extra mechanism to monitor assets, because (IIUC) never get in the  `Syncer#@pending_updates` array (so, showing the files is really a stretch goal for this task). 

```
09:55:05 Pushed » 'theme-app-extension' to a draft
   - blocks/rockets.liquid
   - assets/base.css
09:55:28 Pushed » 'theme-app-extension' to a draft
   - blocks/rockets.liquid
```

### PR DESCRIPTION
**Pull request by ulgut**
### WHY are these changes introduced?

Fixes #0000 

### Update checklist

- [ ] I've added a CHANGELOG entry for this PR (if the change is public-facing)
- [ ] I've considered possible cross-platform impacts (Mac, Linux, Windows).
- [ ] I've left the version number as is (we'll handle incrementing this when releasing).
- [ ] I've included any post-release steps in the section above (if needed).

### CODE REVIEW

File: lib/shopify_cli/theme/extension/syncer/extension_serve_job.rb [hunk 1]
```diff
@@ -2,16 +2,19 @@
...
 require "shopify_cli/partners_api"
+require "shopify_cli/thread_pool/job"
 
 module ShopifyCLI
   module Theme
     module Extension
       class Syncer
         class ExtensionServeJob < ThreadPool::Job
-          POLL_FREQUENCY = 1 # second
+          POLL_FREQUENCY = 0.5 # second
```
Comment (jessetuglu) [inline lib/shopify_cli/theme/extension/syncer/extension_serve_job.rb:12]: I figured it would be alright to lower this to 500ms

**Review by karreiro — APPROVED** [review]
Thank you, @jessetuglu! :)
(as we talked on Slack, I've rebase the branch and changed the color of files to blue).

File: lib/shopify_cli/theme/extension/syncer/extension_serve_job.rb [hunk 2]
```diff
@@ -62,6 +69,20 @@ def perform!
...
+
+          def success_message(project)
+            "#{timestamp} {{green:Pushed}} {{>}} {{blue:'#{project}'}} to a draft"
+          end
+
+          def error_message(project)
+            "#{timestamp} {{red:Error}}  {{>}} {{blue:'#{project}'}} could not be pushed:"
+          end
+
+          def print_list(items, func)
```
[the remark this answers is not in the document]
Comment (karreiro) [reply lib/shopify_cli/theme/extension/syncer/extension_serve_job.rb:81]: Good catch! 🚀

**Review by karreiro — APPROVED** [review]
Thank you, @jessetuglu! Awesome work! :)

I've found only a minor issue:
image

When we save multiple changes in a file, the paths get duplicated.

But I believe we can solve this on `main` :)

*Edit* Resolved

**Review by mgmanzella — APPROVED** [review]
nice!

**Pull request merged**